features/config-is-true.feature

```diff
@@ -0,0 +1,56 @@
+Feature: Determine whether the value of a constant or variable defined in wp-config.php and wp-custom-config.php files is true.
+
+  Scenario: Get the value of a variable whose value is true
+    Given a WP install
+
+    When I run `wp config set WP_TRUTH true`
+    Then STDOUT should contain:
+    """
+    Success:
+    """
+    When I try `wp config is_true WP_TRUTH`
+    Then STDOUT should be empty
+    Then STDERR should be empty
+    And the return code should be 0
+
+    When I run `wp config set WP_TRUTH "true"`
+    Then STDOUT should contain:
+    """
+    Success:
+    """
+    When I try `wp config is_true WP_TRUTH`
+    Then STDOUT should be empty
+    Then STDERR should be empty
+    And the return code should be 0
+
+    When I run `wp config set WP_FALSE_STRING "false"`
+    Then STDOUT should contain:
+    """
+    Success:
+    """
+    When I try `wp config is_true WP_FALSE_STRING`
+    Then STDOUT should be empty
+    Then STDERR should be empty
+    And the return code should be 0
+
+    When I run `wp config set WP_STRING "foobar"`
+    Then STDOUT should contain:
+    """
+    Success:
+    """
+    When I try `wp config is_true WP_STRING`
+    Then STDOUT should be empty
+    Then STDERR should be empty
+    And the return code should be 0
+
+  Scenario: Get the value of a variable whose value is not true
+    Given a WP install
+
+    When I run `wp config set WP_FALSE false --raw`
+    Then STDOUT should contain:
+    """
+    Success:
+    """
+    When I try `wp config is_true WP_FALSE`
+    Then STDOUT should be empty
+    And the return code should be 1
```

Comment:
- [ ] Update tests is_true to is-true

## TESTS TO DEFINITELY ADD
* constant with value "0" is true (assert intention with boolval for future developers)
* constant DNE

## TESTS TO POSSIBLY ADD
* variable value true
* variable value false
* constant and variable with same name fails
* constant and variable with same name passes when flag included
* constant defined in a separate file included in wp-config (this works for `get` but not `has`)— skip if inconvenient to scaffold the test.

Reply:
Still would like as a minimum `constant DNE`

Reply:
Many tests added!